src/themes/ec/variables/_typography.scss

```diff
@@ -199,20 +199,60 @@ $typography: (
   ),
   microcopy: (
     '2xs': (
-      'size': var(--fs-2xs),
-      'line-height': var(--lh-3xs),
+      'mobile': (
```

Comment:
i don't get it, microcopy have the same values for all the viewports, why would we generate media queries to duplicate the same values, i wonder...